lib/Config.js

```diff
@@ -629,6 +630,14 @@ class Config extends EventEmitter {
                 if (Array.isArray(config.kmip.transport)) {
                     this.kmip.transport = config.kmip.transport.map(t =>
                         this._parseKmipTransport(t));
+                    if (config.kmip.retries) {
+                        assert(typeof config.kmip.retries === 'number',
+                            'bad config: KMIP Cluster retries must be a number');
+                        assert(config.kmip.retries <= this.kmip.transport.length - 1,
```

Comment:
Is it a good idea to assert in this case?
Maybe we could overwrite with `this.kmip.transport.length - 1` and `log.warn` or `log.error`?

Reply:
Also why `-1`, probably a dumb question.

Reply:
Also why `-1` here, probably a dumb question

Reply:
I guess it's because 1 original try +  (this.kmip.transport.length - 1) retries = 1 try for each host.

Reply:
Ok, makes sense.

Reply:
There the check [d7b2328](https://github.com/scality/Federation/pull/6197/commits/d7b2328e68b7ca9ef5d59a03bcd570df8f008b33)